RELEASE_PROCESS.md

```diff
@@ -30,38 +30,86 @@ In order to release, you will need the following accounts/permissions:
 ### Informing everyone of a pending release
 
 - [ ] Announce your intention to drive the release to #cft-announce and #chef-server on slack.
+CLARIFICATION from @tas:
+    we can probably move the cft announce to be post promote
+    just copy the discourse post link to that channel to make sure folks see it
+    TODO: get the discourse link he was referring to.
 
 ### Getting the build to be released into current with a minor/major version bump
 
-- Update the version file to the version to be released if minor version needs to be bumped rather than just the patch.
+- Create a new branch from the latest master branch ? <----
+- Update the contents of the file VERSION to the version to be released if minor version needs to be bumped rather than just the patch [what about major version?].
+For updating just the patch verion, [insert 'skip this section, and' ?] refer the the section below on #Preparing for the release
 - Update the release notes with the version, date and context.
-- Run all the tasks for update from FrequentTasks.doc
-- Make sure the omnibus build is into current channel.
+QUESTION: here? https://github.com/chef/chef-server/wiki/Pending-Release-Notes
+- Update CHANGELOG.md ? <-----
+- Run all the tasks for update from dev-docs/FrequentTasks.md. [i need a walkthrough of this] <-----
+QUESTION: is this just updating ruby Gemfile.lock and erlang rebar.lock files? just delete the locks and regenerate? no updating of particular deps by hand?
+QUESTION: just make a new branch off of master to do this?
+- Make sure the omnibus build is into current channel. [explain?] <-----
   (This is triggered by expeditor once the build and tests in buildkite go through ok once a commit is merged to master)
-- Make sure the habitat builds are passing.
-- For updating just the patch verion refer the the section below on #Preparing for the release
+QUESTION: when/where does the merge to master happen? where is this step?
+One approach is to enter the following into a bash shell, where _version_ is the version number of the new release:
+```
+$ mixlib-install download chef-server -c current -a x86_64 -p ubuntu -l 16.04 -v <version>
+```
+Starting download https://packages.chef.io/files/current/chef-server/14.1.0/ubuntu/16.04/chef-server-core_14.1.0-1_amd64.deb
+- Make sure the habitat builds are passing. [pipeline? and when/where is this build kicked off?] <-----
+Chef / [chef/chef-server:master] habitat/build / master 
+Monitor the chef-server-notify slack channel for current progress ? <-----
+
+QUESTION: at some point @tas ran a build through the Chef / [chef/chef-server:master] omnibus/release pipeline.  where is this step represented?  explain.
```

Comment:
happens automatically on merge.

Reply:
I will put this somewhere appropriate.  Is this sentence correct?

"Automatically upon merge, a build is kicked-off on the Chef / [chef/chef-server:master] omnibus/release pipeline."